usr/local/www/diag_logs_settings.php

```diff
@@ -187,10 +189,8 @@ function is_valid_syslog_server($target) {
 $pgtitle = array(gettext("Status"), gettext("System logs"), gettext("Settings"));
 $closehead = false;
 include("head.inc");
-
 ?>
 
-
 <script type="text/javascript">
```

Comment:
these Javascripts should be removed in favor of ```$input->toggles()```; put all toggled settings in a separate section for that